ctapipe/__main__.py

```diff
@@ -0,0 +1,63 @@
+from traitlets.config import Application
+
+from .version import __version__
+
+
+class MainTool(Application):
+    """
+    Main entry point for ctapipe, provides other tools as subcommands
+    """
+
+    name = "ctapipe"
+    version = __version__
+
+    subcommands = {
+        "process": (
+            "ctapipe.tools.process.ProcessorTool",
+            "ctapipe event-wise data processing",
+        ),
+        "apply-models": (
+            "ctapipe.tools.apply_models.ApplyModels",
+            "Apply trained machine learning models",
+        ),
+        "train-energy-regressor": (
+            "ctapipe.tools.train_energy_regressor.TrainEnergyRegressor",
+            "Train telescope-type-wise energy regression models",
+        ),
+        "train-particle-classifier": (
+            "ctapipe.tools.train_particle_classifier.TrainParticleClassifier",
+            "Train telescope-type-wise binary classficiation models",
+        ),
+        "train-disp-regressor": (
+            "ctapipe.tools.train_disp_regressor.TrainDispRegressor",
+            "Train telescope-type-wise disp regression models for direction reconstruction",
+        ),
+        "info": (
```

Comment:
missing `"fileinfo"` tool

Reply:
I just pushed a few commits to convert `ctapipe-fileinfo` into a `Tool` and include it in this list. So this should be fixed now.